tests/webidl/post.js

```diff
@@ -1,3 +1,7 @@
+(async function main(){
+// The test setup is the same whether or not MODULARIZE is used, in which
+// `TheModule` is a promise. This handles both cases.
+TheModule = await Promise.resolve(TheModule);
```

Comment:
this change to the test worries me. it makes it hard to see if this is necessary or not in the non-MODULARIZE case, so people looking at the test for an example may be confused. can we split it out so it's just for MODULARIZE, perhaps using a different `post.js` then?

Reply:
Alright, I re-worked it to show that it's not necessary in every case a little more clearly.